explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+
+
+With a TD, clients are informed of the choices they can make when interacting with Things such reading a temperature value or switching on a lamp. This can drive applications in the same way that Web HTML pages allow users to
+make choices when navigating the Web and drive Web applications
+through links and forms. Therefore, the TD can be also seen as the ‘index.html’ for Things. The TD itself is serialized in JSON-LD 1.1. 
+
+TDs are designed to provide all the information needed to interact with a Thing. Specifically, it provides answers to questions like those shown in the following figure.
+
+
+![How to Interact with IoT Devices](images/explainer_td.png)
+
+
+## Getting started
+
+Lets assume there is a smart lamp that offers an interface with following characteristics:
+
+| |  ||  
+--- | --- |  --- |
+|Name|MyLampThing| 
+|Protocol |HTTPS| 
+|Security |Basic authentication| 
+|Local server address |192.168.2.134| 
+|Content type |application/json| 
+|Data / function offerings: |status (type: string) | Resource (GET): /status
+|  |toggle | Resource (POST): /toggle
+|  |overheating (type: string) | Resource (GET & longpoll): /oh
+
+
+Based on such information a Thing Description can be simple designed in the following way
+
+```json
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    "title": "MyLampThing",
+    "securityDefinitions": {
+        "basic_sc": {"scheme": "basic", "in": "header"}
+    },
+    "security": "basic_sc",
+    "properties": {
+        "status": {
+            "type": "string",
+            "forms": [{"href": "https://192.168.2.134/status"}]
+        }
+    },
+    "actions": {
+        "toggle": {
+            "forms": [{"href": "https://192.168.2.134/toggle"}]
+        }
+    },
+    "events":{
+        "overheating":{
+            "data": {"type": "string"},
+            "forms": [{
+                "href": "https://192.168.2.134/oh",
+                "subprotocol": "longpoll"
+            }]
+        }
+    }
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**. 
+
+Each Thing and its data and functions offerings that is possible via the interface can be classified in those affordances. Sensor and/or parameter data are considered as **properties**. Functions like on/off, dimming, etc. are seen as **actions**. Data events, streams etc.  are considered as **events**. 
+
+In this context, the MyLampThing example above would result in declaring status a property, toggling an action, and overheating an event. 
+
+Now, by reading the above example TD,  one can obtain knowledge about the Thing named `MyLampThing`, including the following:
```

Comment:
by reading -> by processing
one -> a client

BTW it's not really clear who or what the "client" is, maybe this can be defined earlier.  A client may be a person reading documentation, or it may be a machine wanting to interact with the Thing.

Reply:
done